draft-ietf-quic-tls.md

```diff
@@ -1197,6 +1197,64 @@ TLS ClientHello.  The server MAY retain these packets for later decryption in
 anticipation of receiving a ClientHello.
 
 
+## Retry Packet Integrity {#retry-integrity}
+
+Retry packets (see the Retry Packet section of {{QUIC-TRANSPORT}}) carry a
+Retry Integrity Tag that provides two properties: it allows discarding
+packets that have accidentally been corrupted by the network, and it ensures
+that valid Retry packets cannot be sent by off-path attackers.
```

Comment:
nit: perhaps soften this to, "... , and it helps mitigate off-path attacker Retry packet injection"? (The word "ensures" is a little misleading to me.)

Reply:
Good point, thanks! Fixed.